stable/traefik/values.yaml

```diff
@@ -354,6 +354,9 @@ deployment:
     httpEnabled: false
     httpsEnabled: false
     dashboardEnabled: false
+    httpPort: 80
```

Comment:
These should be optional (commented out by default)

Reply:
fixed